test/test_fakeredis.py

```diff
@@ -2112,6 +2112,37 @@ def test_zadd_incr(r, ch):
     assert zadd(r, 'foo', {'three': 1.0}, incr=True, xx=True, ch=ch) == 4.0
 
 
+def test_zpopmin_empty(r):
+    assert r.zpopmin('foo') == []
+
+
+def test_zpopmin_default_count(r):
+    zadd(r, 'foo', {'one_a': 1})
+    zadd(r, 'foo', {'one_b': 1})
+    zadd(r, 'foo', {'two_a': 2})
```

Comment:
Could do all three steps in one go, but more importantly, it would be better to use data where the elements with the lowest scores are not the ones that are first lexicographically, to ensure that it is really scores being considered.